### PR TITLE
Moved ui to core

### PR DESCRIPTION
**Pull request by elbashandy**
## Summary/Motivation:
Moving `idaes/ui` to `idaes/core/ui`

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/reference_guides/model_libraries/power_generation/unit_models/steam_valve.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Example
   from idaes.core import FlowsheetBlock
   from idaes.models_extra.power_generation.unit_models.helm import HelmValve
   from idaes.models.properties import iapws95
-  from idaes.ui.report import degrees_of_freedom, active_equalities
+  from idaes.core.ui.report import degrees_of_freedom, active_equalities
```
Comment (andrewlee94) [inline docs/reference_guides/model_libraries/power_generation/unit_models/steam_valve.rst:22]: I am surprised to see these being imported from the `ui` tools - are these the same as the functions from `idaes.core.util.model._statistics` and if so should they be imported from there??

**Review by andrewlee94 — DISMISSED** [review]
I had one comment on an import in the documentation, but otherwise I think this looks good.

**Comment by andrewlee94** [comment]
@elbashandy After PR #814 was merged, some conflicts showed up here due to the changes in the UI files.

File: docs/reference_guides/model_libraries/power_generation/unit_models/steam_valve.rst (hunk 1 above)
Comment (elbashandy) [reply docs/reference_guides/model_libraries/power_generation/unit_models/steam_valve.rst]: Great catch. idaes.ui don't have these functions. They are not being used in the given example, so I chose to remove that line from the documentation

**Comment by elbashandy** [comment]
@andrewlee94 I updated the branch. This is ready for review

**Review by andrewlee94 — APPROVED** [review]

**Review by dangunter — APPROVED** [review]
Looks good

**Pull request merged**